### ISSUE TITLE
Unexpected <!--{cke_protected}> & <!--[if gte vml 1]> comments added

### ISSUE THREAD

**Issue by usmannoor10p** [ckeditor/ckeditor4#5217]
## Type of report

Bug

## Provide detailed reproduction steps

1. Use the following sample word document:
[Table-image.docx](https://github.com/ckeditor/ckeditor4/files/8706215/Table-image.docx)
2. After importing the above word document to Ckeditor, some of the **unwanted** comment `<!--[if gte vml 1]>` is being added to the HTML
3. Once imported whenever I am clicking on any of the table rows this comment `<!--[if gte vml 1]>` is being encoded to `<!--{cke_protected}>` which is causing parsing issues in the HTML.
4. How can we skip/escape this kind of unexpected comment in the Ckeditor? Is there any configuration that Ckeditor 4 has to remove them?
5. Encoded comment HTML:
[Encoded-table.txt](https://github.com/ckeditor/ckeditor4/files/8706332/Encoded-table.txt)
6. Decoded comment HTML:
[Decoded-table.txt](https://github.com/ckeditor/ckeditor4/files/8706333/Decoded-table.txt)

 

### Expected result

`<!--{cke_protected}> & <!--[if gte vml 1]>` should not be added automatically in the HTML

### Actual result

`<!--{cke_protected}> & <!--[if gte vml 1]>` are added in the HTML

## Other details

* Browser: Version 101.0.4951.54 (Official Build) (64-bit)
* OS: Windows 10 Version 1090 (OS Build 18363.836)
* CKEditor version: 4.14.1

**Comment by jacekbogdanski** [comment]
@usmannoor10p could you share editor configuration using one of the online editors suggested in issue template?

* CodePen: https://codepen.io/karoldawidziuk/pen/LYzJvdx
* JSFiddle: https://jsfiddle.net/Kratek_95/nhwe5uLq
* JSBin: https://jsbin.com/gubepar/edit?html,js,output
* StackBlitz: https://stackblitz.com/edit/ckeditor4-bug-report

I can't reproduce the issue using default, minimal editor configuration (a screencast would be also welcome, if that issue occurs when using editor).

**Comment by usmannoor10p** [comment]
@jacekbogdanski thanks for your quick response. Let me try to add an example of any of the given environments. Meanwhile please go through the attached **Encoded and Decoded** Table files. You will be able to see the comments.

**Comment by usmannoor10p** [comment]
[table_cke_protected.zip](https://github.com/ckeditor/ckeditor4/files/8713824/table_cke_protected.zip)

@jacekbogdanski above is the example, in which you will be able to see the `{cke_protected}` tag being converted.

**Comment by KarolDawidziuk** [comment]
Hello, @usmannoor10p, thank you for the provided example, and sorry for the late reply.

I can't reproduce this issue, I checked the `.docx` file and your sample and everything looks good. 

Comments were not added after I paste content from `.docx` file and in your provided example the comments was not replaced when I clicked on the table rows. Checked on Windows and macOS.

But I want to be sure that I fully understand your issue.

There is an issue with adding and replacing `<!--[if gte vml 1]>` by `<!--{cke_protected}>` which causes the parsing issues? 

Can I ask you for a short video that will show your issue? I would like to be sure that I have not missed something.
